pkgs/applications/misc/streamdeck-ui/default.nix

```
@@ -3,45 +3,30 @@
 , fetchFromGitHub
 , fetchpatch
 , copyDesktopItems
-, wrapQtAppsHook
 , writeText
 , makeDesktopItem
 , xvfb-run
-, qt5
+, qt6
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "streamdeck-ui";
-  version = "2.0.6";
+  version = "3.0.1";
 
   src = fetchFromGitHub {
     repo = pname;
```

```suggestion
    repo = "streamdeck-linux-gui";
```

Seems to has changed as well (https://github.com/streamdeck-linux-gui/streamdeck-linux-gui) wouldn't change the package-name